xdm/config/__init__.py

```diff
@@ -17,7 +17,6 @@ def __init__(self, *args, **kwargs):
         super(Config, self).__init__()
 
         self.load_defaults()
-        self.load_arguments(self.arguments)
```

Comment:
this is repeated 2 lines later - is it here as a left over from previous versions, or does it do something special?